flask_restplus/api.py

```diff
@@ -65,6 +65,7 @@ class Api(object):
     :param str contact: A contact email for the API (used in Swagger documentation)
     :param str license: The license associated to the API (used in Swagger documentation)
     :param str license_url: The license page URL (used in Swagger documentation)
+    :param str host: The host that the api is running on.  Not always the localhost, thought it is the default.
```

Comment:
There is many closed issue on the host subject (ex: #132, #155, #128, #54).
You should have to enter it manually as Flask already handle this properly. If you have to specify the host elsewhere from the `SERVER_NAME` setting, it's propably that you're lacking the [ProxyFix](http://flask.pocoo.org/docs/0.10/deploying/wsgi-standalone/#proxy-setups)

I should had a dedicated section in the documentation.